index.dd

```diff
@@ -392,15 +392,16 @@ $(DIVC frontpage-orgs center,
 )
     <div id="frontpage-orgs-more">
         <i class="fa fa-angle-double-right" aria-hidden="true"></i>
+        $(LINK2 $(ROOT_DIR)areas-of-d-usage.html, Current D use)
+        $(BR)
+        <i class="fa fa-angle-double-right" aria-hidden="true"></i>
```

Comment:
I found this interesting explanation on StackOverflow:

> Usage is how something is used; the fact of something being used is use; the degree to which something is used is utilization.

http://english.stackexchange.com/questions/34712/use-vs-usage